frontend/src/static/js/v_summary.js

```diff
@@ -91,17 +91,20 @@ window.vSummary = {
       return totalCommits / totalCount;
     },
     avgContributionSize() {
-      let totalLines = 0;
-      let totalCount = 0;
-      this.filtered.forEach((repo) => {
-        repo.forEach((user) => {
-          if (user.totalCommits > 0) {
-            totalCount += 1;
-            totalLines += user.totalCommits;
-          }
+      if (typeof meanContributionSize === 'undefined') {
```

Comment:
why need to do this, cannot just do everything without the if and `return totalLines/totalCount;`?

Reply:
> why need to do this, cannot just do everything without the if and return totalLines/totalCount;?

To improve the response time. Since its a value that do not change, we should not compute the value every single instance.